server/src/main/java/io/crate/execution/engine/indexing/ColumnIndexWriterProjector.java

```diff
@@ -76,6 +79,7 @@ public ColumnIndexWriterProjector(ClusterService clusterService,
                                       @Nullable Symbol routingSymbol,
                                       ColumnIdent clusteredByColumn,
                                       List<Reference> columnReferences,
+                                      @NotNull List<Reference> partitionedByColumns,
                                       List<Input<?>> insertInputs,
```

Comment:
this comes from `DocTableInfo.partitionedByColumns` which afaik is not null, also all streaming and all handling components receiving it doesn't do null checks.

Entry point is `DocIndexMetadata.getPartitionedBy`, looks like safe to say it's not null?